### ISSUE TITLE
Server-side MPU is hardcoded to create a directory prefix of only a single character

### ISSUE THREAD

**Issue by dekobon** [joyent/java-manta#451]
MPU objects are organized in a directory structure that looks like as follows:

```
007/
00b/
017/
```
For our purposes, we will call these directories "prefix directories".

Inside one of those directories, you will see another directory like:
```
01743291-e247-69ab-bdbc-a9d902f52c13/
```

This directory name represents a MPU id. Manta is configurable (in server settings) to allow for prefix directories of different lengths. The length of the prefix directory can be determined by the last character (as a digit) in the MPU id. For the example above, the last character of `01743291-e247-69ab-bdbc-a9d902f52c13` would be `3`, so the prefix directories are all 3 characters.

The MPU code in the Java Manta SDK was written at a time when the design of MPU was hard coded to a single character prefix directory.

In summary, operations like abort MPU will not function because a valid prefix directory cannot be created.
